index.html

```diff
@@ -880,13 +880,29 @@ <h2>
 
 </pre>
     </section>
+    <section class='informative'>
+      <h2>
+        Privacy and Security Considerations
+      </h2>
+
+     
+      <section>
+        <h3>Access to aspects of a user’s local computing environment</h3>
+        <p>
+          The screen orientation type and angle of the device can be accessed with the API specified in this document,
+          and can be a potential fingerprinting vector. However, using this information is very difficult compared to other fingerprinting information. 
+          Aggregating information derived from this API is not very helpful in fingerprinting. 
+          In particular the <a href="https://w3c.github.io/deviceorientation/spec-source-orientation.html">DeviceOrientation Event Specification</a> provides a far more detailed version of the same information. 
```

Comment:
Same, all this stuff can go. Blaming another spec for being more leaky doesn't help this one :)